.env.example

```diff
@@ -424,6 +424,17 @@ MODULE_litecoin-mweb_NODES[]=http://login:password@127.0.0.2:1234/
 MODULE_litecoin-mweb_REQUESTER_TIMEOUT=60
 MODULE_litecoin-mweb_REQUESTER_THREADS=12
 
+#######################
+## Moonbeam Main Module
```

Comment:
The module is `moonbeam`, but the branch name is `moonriver`, something is wrong.